src/main/java/org/apache/accumulo/testing/continuous/UndefinedAnalyzer.java

```diff
@@ -247,6 +247,11 @@ TabletAssignment findMostRecentAssignment(String row, long time1, long time2) {
   /**
    * Class to analyze undefined references and accumulo logs to isolate the time/tablet where data
    * was lost.
+   * 
+   * @param args
+   *          the command line arguments
+   * @throws Exception
+   *           if an exception occurs
```

Comment:
I think you have your warnings set weird. We normally exclude missing standard javadocs from warnings, because it forces us to put things in like this, which doesn't add any value from what you can see already from the method signature.

Reply:
For the record - I saw the errors in the maven build - it was not a setting in my IDE.

If the update to the configuration eliminates the warning, this PR can be rejected.

Reply:
Verified that the changes in #180 eliminates the warnings - the changes in this PR are not necessary to remove the warnings when building with maven.

Reply:
Cool, but if you want to keep any of the updated javadocs, feel free to do so. Some of these could still be useful for their explanatory power, and not just to eliminate warnings.

Reply:
```suggestion
```

Reply:
The comment 
```Some of these add value, but a lot of them just repeat information already shown in the method signatures, which isn't useful. ```
is subjective - the changes I added were from a simple reading of the signature / or the code, not from a deep understanding - so I could argue that they add all add very little information.